src/screens/Customers/ShowCustomers.res

```diff
@@ -55,6 +55,58 @@ module CustomerInfo = {
   }
 }
 
+module CustomerDetails = {
+  open GlobalSearchBarUtils
+  open GlobalSearchTypes
+  open APIUtils
+  @react.component
+  let make = (~id) => {
+    let getURL = useGetURL()
+    let fetchData = APIUtils.useUpdateMethod()
+    let (searchResults, setSearchResults) = React.useState(_ => [])
+    let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Loading)
+
+    let getSearchResults = async () => {
+      setScreenState(_ => PageLoaderWrapper.Loading)
+      try {
+        let url = getURL(~entityName=GLOBAL_SEARCH, ~methodType=Post, ())
+
+        let body = [("query", id->JSON.Encode.string)]->LogicUtils.getJsonFromArrayOfJson
+
+        let response = await fetchData(url, body, Post, ())
+
+        let remote_results = response->parseResponse
+
+        let data = {
+          local_results: [],
+          remote_results,
+          searchText: id,
+        }
+
+        let (results, _) = data->SearchResultsPageUtils.getSearchresults
+
+        setSearchResults(_ => results)
+        setScreenState(_ => PageLoaderWrapper.Success)
+      } catch {
+      | Exn.Error(e) =>
+        let err = Exn.message(e)->Option.getOr("Failed to Fetch Customer Details")
+        setScreenState(_ => PageLoaderWrapper.Error(err))
```

Comment:
Please check whether we need set the screen state to error.I hope its not required.show the page only if the data is there or show no data found